pages/competition/[slug]/index.vue

```diff
@@ -118,6 +87,13 @@ const influenceRanking = computed(() => {
   };
 });
 
+const canShowInfluence = computed(() => {
+  return (
+    competition?.value?.has_influence &&
```

Comment:
Se já tiver acabado, deve mostrar também o ranking, né?
 
 competition?.value?.has_influence && influenceRanking.value.contents.length > 0

Reply:
o que acontece se não tiver acabado e não tiver influencias ainda??

Reply:
aparece!

primeira parte do AND checa se tem influencia na copa
segunda parte checa se ou tem alguem no ranking ou a competicao nao acabou

ou seja, aparece tanto qnd acabado (dado que tem ao menos 1 pessoa no ranking) e se nao tiver acabado